src/words.jl

```diff
@@ -86,11 +96,11 @@ end
 Returns the length of longest common prefix of two words (and simultaneously
 the index at which the prefix ends).
 """
-function longestcommonprefix(u::Word, v::Word)
+function longestcommonprefix(u::Union{Word, SubWord}, v::Union{Word, SubWord})
```

Comment:
Maybe it would be a good idea to call this function using an abbrev.? Maybe `lcp`/`LCP`? I know it is not that important, but I cannot stop imaging `greatestcommondivisor` and it makes me feel dizzy.

Reply:
yeah, I might be leaning towards it as well. Despite the fact that I asked for the wordy name in the first place :laughing: